athena-elasticsearch/src/main/java/com/amazonaws/athena/connectors/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -186,10 +188,20 @@ public ListTablesResponse doListTables(BlockAllocator allocator, ListTablesReque
                 .filter(index -> !index.startsWith("."));
 
         //combine two different data sources and create tables
-        List<TableName> tableNames = Stream.concat(indicesStream, getDataStreamNames(client))
-                .map(tableName -> new TableName(request.getSchemaName(), tableName))
-                .collect(Collectors.toList());
-        return new ListTablesResponse(request.getCatalogName(), tableNames, null);
+        Stream<TableName> tableNamesStream = Stream.concat(indicesStream, getDataStreamNames(client)).sorted()
+                .map(tableName -> new TableName(request.getSchemaName(), tableName));
```

Comment:
you can move this `.map(tableName -> new TableName(request.getSchemaName(), tableName));` part to line 201 prevent from creating tablename if we skip